nilearn/regions/signal_extraction.py

```diff
@@ -15,8 +15,146 @@
 from .. import masking
 from ..image import new_img_like
 
+INF = 1e-9
+
+
+def __check_labels(labels_img, target_shape, target_affine):
```

Comment:
Could you rename this `_check_labels` with a single `_` as for any other private function in nilearn?

Reply:
sure, thanks!